src/custom-types.json

```diff
@@ -143,14 +143,26 @@
   "application/vnd.apple.pages": {
     "compressible": false,
     "extensions": ["pages"],
+    "sources":[
+      "https://docs.alfresco.com/5.2/references/valid-transformations.html",
+      "https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html",
```

Comment:
Neither of these look like a Primary source (neither seem to be from Apple or something related to Apple) and I don't see on either one anywhere that specially says that the MIME application/vnd.apple.pages === .pages file extension. Can you clarify this, please?